scripts/.env

```diff
@@ -1,2 +1,2 @@
-NEXUS_VERSION=3.33.0
-NEXUS_PORT=8081
+export NEXUS_VERSION=3.37.0
```

Comment:
please change the documentation of the provider to the new version

Reply:
`in file docs/index.md`

Reply:
👍